core/network/network-api/src/commonMain/kotlin/com/droidknights/app/core/network/api/DroidknightsNetwork.kt

```diff
@@ -0,0 +1,11 @@
+package com.droidknights.app.core.network.api
+
+import io.ktor.util.reflect.TypeInfo
+import io.ktor.util.reflect.typeInfo
+
+interface DroidknightsNetwork {
```

Comment:
```suggestion
interface DroidKnightsNetwork {
```

(사소) 파스칼 케이스 적용해보면 좋을 것 같아요